src/components/resumes/Details.jsx

```diff
@@ -34,100 +51,112 @@ function Details({ resumeId }) {
       setShowEdit("Edit");
     }
   };
-  if (error) return <div>ERROR!!! Resume not found</div>;
+  if (error) return <NotFound />;
   return data ? (
-    <Grid className={classes.root} container spacing={5}>
-      <Grid className={classes.margin} item xs={12}>
-        <Typography color="primary" variant="h4">
-          {data.name} {data.username}
-        </Typography>
-        <div>creation date</div>
-        <div>last update</div>
-      </Grid>
-
-      <Grid className={classes.margin} item xs={18}>
-        <TextField
-          InputProps={{
-            readOnly: readOnly,
-          }}
-          defaultValue={data.name}
-          label="First Name"
-          variant="outlined"
-        />
-        <TextField
-          InputProps={{
-            readOnly: readOnly,
-          }}
-          defaultValue={data.username}
-          label="Last Name"
-          variant="outlined"
-        />
-        <TextField
-          InputProps={{
-            readOnly: readOnly,
-          }}
-          defaultValue="2017-05-24"
-          label="Birthdate"
-          type="date"
-          variant="outlined"
-        />
-      </Grid>
-      <Grid className={classes.margin} item xs={12}>
-        <TextField
-          InputProps={{
-            readOnly: readOnly,
-          }}
-          defaultValue={data.email}
-          label="Email"
-          variant="outlined"
-        />
-        <TextField
-          InputProps={{
-            readOnly: readOnly,
-          }}
-          defaultValue={data.phone}
-          label="Phone"
-          variant="outlined"
-        />
-        <TextField
-          InputProps={{
-            readOnly: readOnly,
-          }}
-          defaultValue={data.address.street}
-          label="Address"
-          variant="outlined"
-        />
-      </Grid>
-      <Grid className={classes.margin} item xs={12}>
-        <TextField
-          InputProps={{
-            readOnly: readOnly,
-          }}
-          defaultValue={data.website}
-          fullWidth
-          label="Summary"
-          multiline
-          rows={4}
-          variant="outlined"
-        />
-      </Grid>
-      <Grid className={classes.margin} item xs={12}>
-        <Button color="primary" onClick={edit} variant="contained">
-          {showEdit}
-        </Button>
-        <Button
-          color="primary"
-          onClick={() => {
-            alert("Back");
-          }}
-          variant="contained"
+    <Grid
+      className={classes.content}
+      style={{
+        bottom: 130,
+        left: "30%",
+        position: "absolute",
+        top: 130,
+      }}
+    >
+      <Paper className={classes.paper}>
+        <form autoComplete="off" className={classes.root} noValidate>
```

Comment:
Could you improve the UI/UX a little? maybe Applying some styles to the fields when they are readonly or on edit mode? 
You could disable them or apply colors...or maybe change the variant of the component. It's up to you to decide :)

Reply:
Please use Formik to validate and handle the form's data.